procedures/crates.md

```diff
@@ -94,6 +94,12 @@ request.
 Recommendations for working with out-of-tree crates will be documented in the rustc-guide, see
 [rust-lang/rustc-guide#285][guide_issue] for progress.
 
+### How should stabilization/semantic changes be handled in out-of-tree crates?
+It is important to involve the language team in any changes in out-of-tree crates that would result
+in stabilization or semantic changes to the language. Submodule changes in PRs to `rust-lang/rust`
+should be labelled appropriately (eg. `relnotes`, `T-compiler`, `T-lang`) just as if the change
+were implemented in `rust-lang/rust` directly.
```

Comment:
and the PR should ideally also explain what is changed in terms that would be understood in rust-lang/rust (maybe that's obvious but maybe not to everyone... ^,-)

Reply:
Added a sentence.